WordPress/Classes/Models/Person.swift

```diff
@@ -90,7 +92,7 @@ struct Person {
             }
         }
         
-        static let roles : [Role] = [.SuperAdmin, .Admin, .Editor, .Author, .Contributor]
+        static let roles = [Role.Admin, .Editor, .Author, .Contributor]
```

Comment:
Good point Eric!. Reason why is: because we don't really want users to set the "Super Admin" role.
I'll document that!